extensions/amp-accordion/1.0/animations.js

```diff
@@ -26,51 +26,58 @@ const COLLAPSE_CURVE = 'cubic-bezier(0.39, 0.575, 0.565, 1)';
  * @return {!UnlistenDef}
  */
 export function animateExpand(content) {
-  return animate(content, () => {
-    const oldHeight = getStyle(content, 'height');
-    const oldOpacity = getStyle(content, 'opacity');
-    const oldOverflowY = getStyle(content, 'overflowY');
+  return animate(
+    content,
+    () => {
+      const oldHeight = getStyle(content, 'height');
+      const oldOpacity = getStyle(content, 'opacity');
+      const oldOverflowY = getStyle(content, 'overflowY');
 
-    // Measure the expanded height. This is relatively heavy with a sync
-    // layout. But no way around it. The hope that the `commitStyles` API
-    // may eventually make this unneeded.
-    setStyles(content, {
-      height: 0,
-      opacity: 0,
-      overflowY: 'auto',
-    });
-    content.hidden = false;
-    const targetHeight = content./*OK*/ scrollHeight;
+      // Measure the expanded height. This is relatively heavy with a sync
+      // layout. But no way around it. The hope that the `commitStyles` API
+      // may eventually make this unneeded.
+      setStyles(content, {
+        height: 0,
+        opacity: 0,
+        overflowY: 'auto',
+      });
+      content.hidden = false;
+      content.classList.add('i-amphtml-animating');
+      const targetHeight = content./*OK*/ scrollHeight;
 
-    // Reset back. The animation will take care of these properties
-    // going forward.
-    setStyles(content, {
-      height: oldHeight,
-      opacity: oldOpacity,
-      overflowY: oldOverflowY,
-    });
+      // Reset back. The animation will take care of these properties
+      // going forward.
+      setStyles(content, {
+        height: oldHeight,
+        opacity: oldOpacity,
+        overflowY: oldOverflowY,
+      });
 
-    const duration = getTransitionDuration(targetHeight);
+      const duration = getTransitionDuration(targetHeight);
 
-    return content.animate(
-      [
-        {
-          height: 0,
-          opacity: 0,
-          overflowY: 'hidden',
-        },
+      return content.animate(
+        [
+          {
+            height: 0,
+            opacity: 0,
+            overflowY: 'hidden',
+          },
+          {
+            height: targetHeight + 'px',
+            opacity: 1,
+            overflowY: 'hidden',
+          },
+        ],
         {
-          height: targetHeight + 'px',
-          opacity: 1,
-          overflowY: 'hidden',
-        },
-      ],
-      {
-        easing: EXPAND_CURVE,
-        duration,
-      }
-    );
-  });
+          easing: EXPAND_CURVE,
+          duration,
+        }
+      );
+    },
+    () => {
```

Comment:
The changes are to add the callback below, and add the class on line 45.  Adding the callback caused the whole function to get indented.